src/store/region-context.tsx

```diff
@@ -65,19 +66,27 @@ export const RegionProvider = ({ children }: RegionProviderProps) => {
 
     useEffect(() => {
         const is_eu_country_ip = eu_countries.includes(user_ip_country)
-        const is_africa = african_countries.includes(user_ip_country)
+        const is_africa =
+            african_countries.includes(user_ip_country) || african_countries.includes(qa_url_region)
         const is_eu_residence = eu_countries.includes(residence)
         const is_eu_location = is_eu_residence || (!residence && is_eu_country_ip)
         const is_eu = isEuDomain() || eu_countries.includes(qa_url_region)
         const is_non_eu = !is_eu
-        const is_cpa_plan = cpa_plan_countries.includes(user_ip_country)
-        const is_latam = latam_countries.includes(user_ip_country)
+        const is_cpa_plan =
+            cpa_plan_countries.includes(user_ip_country) ||
+            cpa_plan_countries.includes(qa_url_region)
+        const is_latam =
+            latam_countries.includes(user_ip_country) || latam_countries.includes(qa_url_region)
         const is_row = !is_eu
         const is_dev = isLocalhost() || isTestlink()
         if (website_status) {
             const { clients_country, p2p_config } = website_status
             setEuCountry(!!isEuCountry(clients_country))
             setP2PAllowedCountry(!!p2p_config)
+            //QA testing purposes
+            p2p_countries.includes(qa_url_region)
+                ? setP2PAllowedCountry(true)
+                : setP2PAllowedCountry(false)
```

Comment:
let's add additional check 

only if `qa_url_region` is available. 

```
 if(qa_url_region) {
    p2p_countries.includes(qa_url_region)
                ? setP2PAllowedCountry(true)
                : setP2PAllowedCountry(false)
}

Reply:
yes. correct. added the check